lint/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
This file should be unnecessary since we have a root-level ignore of all `build` folders.